README.md

```diff
@@ -93,3 +93,63 @@ Understands the following events:
 </pre></td>
   </tr>
 </table>
+
+## Linking local web-workspace-navigator into node_modules
+
+In order to work with non published artifacts of web-workspace-navigator within the test-editor-web, linking can be used.
+Given the following variables, this are the steps to have functional linking in place without modifying any sources.
+TODO: if deemed useful these steps should be put into scripts.
+
+``` shell
+WORKSPACE_NAVIGATOR="~/path/to/project"
+TEST_EDITOR_WEB="~/path/to/project"
+PREFIX=`npm prefix -g`
+```
+
+### bring linking into place
+
+``` shell
+cd $WORKSPACE_NAVIGATOR
+npm link
+cd $TEST_EDITOR_WEB
+npm link @testeditor/workspace-navigator
```

Comment:
… and this has test editor web's dependency on workspace navigator point to the location at which it was previously made available…